src/commands/lint.ts

```diff
@@ -24,6 +24,14 @@ export class LintCommand implements Command {
 
   args = [
     {
+      name: 'input',
```

Comment:
@robdodson I think glob support would be a great follow on to this. By default, given the project entrypoints, we shouldn't be linting bower_components.